sendgrid/helpers/exceptions/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class SendGridException(Exception):
+    def __init__(self, *args, **kwargs):
```

Comment:
We are actually instantiating an object of class here. So it makes sense to keep them.